README.md

```diff
@@ -30,12 +44,41 @@ If you wish to get more information about any command within `rdme`, you can exe
 
 ### Common `rdme` Options
 
-- `--key <string>`: The API key associated with your ReadMe project. You can obtain this from your dashboard, or alternatively if you log in with `rdme login`, we will save your API key to a local configuration file (`~/.config/configstore/rdme-production.json`), saving you the hassle of having to supply this argument on commands that have it.
-- `--version <string>`: Your project version.
+- `--key <string>`: The API key associated with your ReadMe project. Note that most of the commands below require API key authentication, even though the `--key` flag is omitted from the examples. See the [Authentication](#authentication) section above for more information.
+- `--version <string>`: Your project version. See [our docs](https://docs.readme.com/docs/versions) for more information.
+
+### GitHub Actions
```

Comment:
Do you also have to install `rdme` in devDeps if you're using the github action, or no?

Reply:
Nope! You just list it in your workflow YAML file and you're good to go 🚀 

That's also why I have this callout here so people don't install the package for GitHub Action usage:https://github.com/readmeio/rdme/blob/9e895ffc69504b373a9d5593c6d44219d52b479e/README.md#L13